pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -110,7 +111,8 @@ def scale_batch_size(trainer,
 
     # Restore initial state of model
     trainer.checkpoint_connector.restore(str(save_path), on_gpu=trainer.on_gpu)
-    os.remove(save_path)
+    if fs.exists(save_path):
+        fs.rm(save_path)
```

Comment:
should this check whether the trainer is on rank-0?

Reply:
don't know how to check that. can you tell

Reply:
`trainer.is_global_zero`: https://github.com/PyTorchLightning/pytorch-lightning/blob/66ade19d564cc6420a39cb2a64203b5cd21a4b43/pytorch_lightning/trainer/properties.py#L83-L85